js/cid/cid.js

```diff
@@ -1,33 +1,9 @@
+var cid = require('can-cid');
 /**
- * @module {function} can-util/js/cid/cid cid
+ * @module can-util/js/cid/cid
  * @parent can-util/js
- * @signature `cid(object, optionalObjectType)`
- *
- * Get a unique identifier for the object, optionally prefixed by a type name.
- *
- * Once set, the unique identifier does not change, even if the type name
- * changes on subsequent calls.
- *
- * ```js
- * var cid = require("can-util/js/cid/cid");
- * var x = {};
- * var y = {};
- * 
- * console.log(cid(x, "demo")); // -> "demo1"
- * console.log(cid(x, "prod")); // -> "demo1"
- * console.log(cid(y));         // -> "2"
- * ```
- *
- * @param {Object} object The object to uniquely identify.
- * @param {String} name   An optional type name with which to prefix the identifier 
- *
- * @return {String} Returns the unique identifier
+ * @description Deprecated. Use [can-cid] instead.
  */
-var cid = 0;
-module.exports = function (object, name) {
-	if (!object._cid) {
-		cid++;
-		object._cid = (name || '') + cid;
-	}
-	return object._cid;
-};
+console.warn('can-util/js/cid/cid is deprecated, use can-cid instead');
```

Comment:
Ok, interesting. I'm going to remove the warnings here anyway, but we should look at all the other places we use `console.warn` in other can-* repos.